packages/gitbeaker-node/src/GotRequester.ts

```diff
@@ -51,18 +52,29 @@ export function processBody({ rawBody, headers }: { rawBody: Buffer; headers: ob
 }
 
 export async function handler(endpoint, options) {
```

Comment:
I think for got, we just need to increase the retry limit, it handled the other logic internally https://github.com/sindresorhus/got/blob/master/source/core/calculate-retry-delay.ts

Reply:
https://github.com/sindresorhus/got#retry

Reply:
I think that doesn't work, because GitLab doesn't actually send any headers with `Retry-Later`. That's why it doesn't work. Got already has 10 retries by default.